src/main/java/ch/njol/skript/expressions/ExprTool.java

```diff
@@ -106,15 +109,14 @@ public Class<Slot> getReturnType() {
 	}
 
 	@Override
-	public String toString(final @Nullable Event e, final boolean debug) {
+	public String toString(@Nullable Event event, boolean debug) {
 		String hand = offHand ? "off hand" : "";
-		return String.format("%s tool of %s", hand, getExpr().toString(e, debug));
+		return String.format("%s tool of %s", hand, getExpr().toString(event, debug));
```

Comment:
should probably use a SSB here as currently a leading space would occur if not in offhand